styles/styles.less

```diff
@@ -1,5 +1,9 @@
 @import "variables";
 
+atom-pane-container atom-pane div.item-views {
```

Comment:
This is to let the user scroll horizontally when the viewport has a thin width. I'm definitely open to better suggestions on how to achieve this